sqlx-core/src/row.rs

```diff
@@ -176,3 +177,32 @@ pub trait Row: Unpin + Send + Sync + 'static {
     where
         I: ColumnIndex<Self>;
 }
+
+pub fn debug_row<R>(row: &R, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result
```

Comment:
I'm not sure about this being `pub` but given that it's defined in the core crate and used in the database crates I don't think there's a better way. 

Happy to either add docs or a `#[doc(hidden)]`